### PR TITLE
Mafia: Edit Classic, Add Lighthouse, Alignments and Miscellaneous

### PR DESCRIPTION
**Pull request by Snaquaza**

### CODE REVIEW

File: chat-plugins/mafia-data.js [hunk 1]
```diff
@@ -53,12 +55,25 @@ exports.alignments = Object.assign(Object.create(null), {
...
 		// TODO image for cult faction
 		image: `<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-villager.png"/>`,
 	},
+	cultafia: {
+		name: `Cultafia`,
+		plural: `Cultafia`,
+		id: `cultafia`,
+		color: `#600`,
+		memo: [
+			`Factional Communication: If there are other Cult-aligned players, you may PM them during the game.`,
```
Comment (whalemer) [inline chat-plugins/mafia-data.js]: are cultafia separate from normal cults?

File: chat-plugins/mafia-data.js [hunk 2]
```diff
@@ -83,6 +98,59 @@ exports.alignments = Object.assign(Object.create(null), {
...
+		],
+		image: `<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-villager.png"/>`,
+	},
+	pygmee: {
+		name: `Pygmee`,
+		plural: `Pygmees`,
+		id: `pygmee`,
+		color: `#640`,
+		memo: [
+			`Factional Knowledge: If there are other Replicant-aligned players, you know who they are.`,
```
Comment (whalemer) [inline chat-plugins/mafia-data.js]: ahem

File: chat-plugins/mafia-data.js (hunk 1 above)
Comment (Snaquaza) [reply chat-plugins/mafia-data.js]: Yes. (And you can't do Cult Mafia)

File: chat-plugins/mafia-data.js (hunk 2 above)
Comment (Snaquaza) [reply chat-plugins/mafia-data.js]: Actually went through all OSes to see if there were non-Mafia copied alignments. As I commented in the file, I am alright with adding Terrorist / Mermaids as well, but I'd like to get a re-review for those themes first. I've talked with the creator from Mermaids and that one is going to be revamped first anyway.

File: chat-plugins/mafia-data.js [hunk 3]
```diff
@@ -83,6 +98,59 @@ exports.alignments = Object.assign(Object.create(null), {
...
+		color: `#A26`,
+		memo: [
+			`Factional Communication: If there are other Hypnotist-aligned players, you may PM them during the game.`,
+			`Factional Hypnotise: The Hypnotists can hypnotise another player every night. For the next day only, this player gains an additional win-condition of lynching another player specified by the Hypnotists.`,
+			`You are aligned with the <span style="color:#A26;font-weight:bold">Hypnotists</span>. You win when all players without a Hypnotist wincon are eliminated and at least one Hypnotist-aligned player is still alive (or nothing can prevent the same).`,
+		],
+		image: `<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-villager.png"/>`,
+	},
+	// Mermaid theme doesn't seem balanced enough to officially recognise. (Mermaid faction)
+	// Would like to discuss Terrorist theme too. (Terrorist faction)
```
Comment (HoeenCoder) [inline chat-plugins/mafia-data.js]: Remove unneeded comments please

**Pull request merged**